image-rs/src/signature/mechanism/simple/mod.rs

```diff
@@ -219,8 +243,13 @@ impl SimpleParameters {
         let sigstore_uri = url::Url::parse(&sigstore)
             .map_err(|e| anyhow!("Failed to parse sigstore_uri: {:?}", e))?;
 
-        let sigs = sigstore::get_sigs_from_specific_sigstore(sigstore_uri).await?;
+        let mut sigstore_sigs = sigstore::get_sigs_from_specific_sigstore(sigstore_uri).await?;
+        sigs.append(&mut sigstore_sigs);
 
         Ok(sigs)
     }
 }
+
+pub async fn new_sigs_vec() -> Vec<Vec<u8>> {
```

Comment:
Why not we directly use
```rust
let mut sigs: Vec<Vec<u8>> = Vec::new();
```
in `fn get_signatures()`?

Reply:
When I was trying to build the code with my changes before I got an error, and I thought the cause of the error was the async function `allows_image` was returning a `Vec<Vec<u8>>` that wasn't created itself by an async function, so i added the `new_sigs_vec` async function you saw.

I made the change you suggested and my code still builds.

So the error I got before must have been caused by another error and my original 'fix' wasn't necessary.